### ISSUE TITLE
Scheduler raises a warning

### ISSUE THREAD

**Issue by jianguoYao** [nicolas-chaulet/torch-points3d#487]
UserWarning: The epoch parameter in `scheduler.step()` was not necessary and is being deprecated where possible. Please use `scheduler.step()` to step the scheduler. During the deprecation, if epoch is different from None, the closed form is used instead of the new chainable form, where available. Please open an issue if you are unable to replicate your use case: https://github.com/pytorch/pytorch/issues/new/choose.
  warnings.warn(EPOCH_DEPRECATION_WARNING, UserWarning)
**why this warnings?**

**Comment by nicolas-chaulet** [comment]
Because we haven't updated the way we handle learning rate and batch norm schedulers. We would have to do that at some point...
